go/client/logclient_test.go

```diff
@@ -229,8 +229,8 @@ func TestAddChainRetries(t *testing.T) {
 	const leeway = time.Millisecond * 100
 	const leewayRatio = 0.1 // 10%
 	const instant = time.Millisecond
-	const fiveSeconds = time.Second * 5
-	const sevenSeconds = time.Second * 7 // = 1 + 2 + 4
+	const five = time.Second * 5
+	const seven = time.Second * 7 // = 1 + 2 + 4
```

Comment:
I'd just remove these variables and put the values directly into the test definitions below. Having them as variables doesn't seem to gain us anything. On a related note, deadlineLength and retryAfter ought to be `time.Duration`s too.